source/extensions/upstreams/http/config.h

```diff
@@ -53,6 +53,9 @@ class ProtocolOptionsConfigImpl : public Upstream::ProtocolOptionsConfig {
   bool use_http2_{};
   bool use_http3_{};
   bool use_alpn_{};
+  using FiltersList = Protobuf::RepeatedPtrField<
```

Comment:
Totally your call, but if this type is only used on the line below, maybe there's not much value in the "using"? (Or maybe it's somehow used elsewhere?) Anyway, totally your call. This is fine as is.